test/test_scaling.jl

```diff
@@ -214,15 +214,18 @@ close(ds)
 add_offset = -1.0
 scale_factor = 0.1
 
-attributes = Dict("add_offset" => add_offset,"scale_factor" => scale_factor)
-
-
 
 for Tbase in (UInt8, Int8, Float32, Float64, Int64, Char, String)
     for _maskingvalue in (NaN,NaN32,missing,42,nothing,mysingleton)
-        local fname, data, fv, ds, ncv, T, varattrib
+        local fname, data, fv, ds, ncv, T, varattrib, attributes
         fname = tempname()
         T = promote_type(Tbase,typeof(_maskingvalue))
+        if T == Float32
```

Comment:
I update these test according to https://github.com/JuliaGeo/CommonDataModel.jl/issues/33